### PR TITLE
[stable/kube-slack] add more options for kube-slack

### PR DESCRIPTION
**Pull request by skomma**
#### What this PR does / why we need it:
This PR adds more options implemented in kube-slack.
(Available options are described in [kube-slack documentation](https://github.com/wongnai/kube-slack#installation))

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by rimusz** [comment]
`Configuration` part with all listed values would be nice to have

**Comment by skomma** [comment]
@rimusz

Thank you for your review.
I added all configurable values in README.md.

**Comment by skomma** [comment]
@seanknox Could you please have a look?

**Comment by skomma** [comment]
@rimusz Sorry, the `appVersion` field is updated on master branch, so please have a look again.

**Comment by skomma** [comment]
@rimusz I updated chart version.

**Comment by skomma** [comment]
@rimusz Thank you for lgtm.
If possible, could you assign another reviewer other than @seanknox ?
(He seems not to be online on GitHub so much...)

File: stable/kube-slack/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kube-slack.fullname" . }}
+  labels:
+    app: {{ template "kube-slack.name" . }}
+    chart: {{ template "kube-slack.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
```
Comment (davidkarlsen) [inline stable/kube-slack/templates/configmap.yaml:10]: would it be better / more flexible and future-proof to just copy a block of values into the configmap? So you don't have to copy and predefine one and one value?

Comment (davidkarlsen) [reply stable/kube-slack/templates/configmap.yaml:10]: @skomma ?

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/kube-slack/templates/configmap.yaml (hunk 1 above)
Comment (skomma) [reply stable/kube-slack/templates/configmap.yaml:10]: Sorry, I'll make the PR again, including the requested changes!!

**Pull request closed without merging**